internal/store/snapshot_schedule.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+//go:generate mockgen -destination=../mocks/mock_backup_snapshot_schedule.go -package=mocks github.com/mongodb/mongocli/internal/store SnapshotScheduleDescriber,SnapshotScheduleUpdater
+
+type SnapshotScheduleDescriber interface {
+	GetSnapshotSchedule(string, string) (*opsmngr.SnapshotSchedule, error)
+}
+
+type SnapshotScheduleUpdater interface {
+	UpdateSnapshotSchedule(string, string, *opsmngr.SnapshotSchedule) (*opsmngr.SnapshotSchedule, error)
+}
+
+// GetSnapshotSchedule encapsulates the logic to manage different cloud providers
+func (s *Store) GetSnapshotSchedule(projectID, clusterID string) (*opsmngr.SnapshotSchedule, error) {
+	switch s.service {
+	case config.OpsManagerService:
```

Comment:
this works in cloud manager, https://docs.cloudmanager.mongodb.com/reference/api/snapshot-schedule/